sectionproperties/pre/library/utils.py

```diff
@@ -32,6 +32,35 @@ def draw_radius(pt: list, r: float, theta: float, n, ccw: bool = True):
         points.append([x, y])
     return points
 
+def draw_radius_60(pt: list, r: float, theta: float, n, ccw: bool = True):
```

Comment:
Why is `n` the only variable with no type hints? Looks like it should always be an int.

`draw_radius_60(pt: list, r: float, theta: float, n: int, ccw: bool = True)`